### ISSUE TITLE
Add param to template path 

### ISSUE THREAD

**Issue by nfultz** [ropensci/postdoc#1]
v minor feature request, would be nice to provide a param (or an env var) to use a different / customized template file. It looks like it would work with URLs also... 

https://github.com/ropensci/postdoc/blob/6ba36abd30676c010b680af61182c8800999987e/R/manual.R#L33

thanks much!

**Comment by jeroen** [comment]
I agree this should be possible. The hardcoded r-universe URLs are not for everyone of course :)

The reason why I haven't added this option yet is that we have not settled on the final structure of the template, including the required `"class"` and `"id"` values of html elements in the template that are used to identify where to insert content. 

I've always found it annoying with pkgdown that our custom themes would break at every release for this reason, so that is why I did not expose it yet.

**Comment by nfultz** [comment]
Other options than class/id magic constants for templates if you are still exploring the solution space:

* jinja style `<p>{{description}}</p>`
* different custom tag: `<p><r:description/></p>` - still technically valid html - may simplify some xpaths ?
* Punt to knitr for rendering - `<p>``r package$description ``<p>`
* XSLT - don't do this though
